kube/services/selenium/selenium-hub-service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: selenium-hub
+  name: selenium-hub
+  namespace: default
+spec:
+  externalTrafficPolicy: Cluster
+  ports:
+  - nodePort: 32026
```

Comment:
do we need to specify a `nodePort`?

Reply:
Not sure, to be honest I just `kubectl get service selenium-hub -o yaml > selenium-hub-service.yaml` and shaved off some k8s metadata.